templates/emails/thankyou.opensource.hbs

```diff
@@ -34,24 +30,40 @@ Subject: Thank you for your {{currency order.totalAmount currency=order.currency
 </p>
 
 <h2>Help us raise more money!</h2>
-<p>Share this URL: <a href="https://opencollective.com/{{collective.slug}}/donate">https://opencollective.com/{{collective.slug}}/donate</a>
+<p>See our activities and transparent budget on
+  <a href="https://opencollective.com/{{collective.slug}}/donate">https://opencollective.com/{{collective.slug}}/donate</a>, and share the link to help us spread the word!
 </p>
 
 {{#if transactionPdf}}
 <h2>📎 Attachments</h2>
 <ul>
   <li>A PDF receipt of your transaction</li>
```

Comment:
<li>A PDF receipt of your contribution to {{collective.name}}</li>

Reply:
Thanks. Correct that in both templates. 😄